conf/config-default.yaml

```
@@ -482,6 +482,7 @@ plugins:                           # plugin list (sorted by priority)
   - limit-count                    # priority: 1002
   - limit-req                      # priority: 1001
   #- node-status                   # priority: 1000
+  - brotli                         # priority: 996
```

I suggest we don't enable the `brotli` plugin by default, since the user may not install the `brotli` library
```suggestion
  #- brotli                         # priority: 996
```